integrationTests/chainSimulator/common.go

```diff
@@ -108,3 +111,31 @@ func SendTransaction(
 
 	return txResult
 }
+
+// IssueFungible will issue a fungible token
+func IssueFungible(
+	t *testing.T,
+	cs ChainSimulator,
+	sender []byte,
+	nonce *uint64,
+	issueCost *big.Int,
+	tokenName string,
+	tokenTicker string,
+	numDecimals int,
+	supply *big.Int,
+) []byte {
+	issueArgs := "issue" +
+		"@" + hex.EncodeToString([]byte(tokenName)) +
+		"@" + hex.EncodeToString([]byte(tokenTicker)) +
+		"@" + hex.EncodeToString(supply.Bytes()) +
+		"@" + fmt.Sprintf("%X", numDecimals) +
+		"@" + hex.EncodeToString([]byte("canAddSpecialRoles")) +
+		"@" + hex.EncodeToString([]byte("true"))
+	txResult := SendTransaction(t, cs, sender, nonce, vm.ESDTSCAddress, issueCost, issueArgs, uint64(60000000))
+	tokenIdentifier := txResult.Logs.Events[0].Topics[0]
+	require.Equal(t, len(tokenTicker)+7, len(tokenIdentifier))
+	require.Equal(t, tokenName, string(txResult.Logs.Events[4].Topics[1]))
+	require.Equal(t, tokenTicker, string(txResult.Logs.Events[4].Topics[2]))
```

Comment:
Don't think this will work forever, the order might change some day;
How about we do something like this?
https://github.com/multiversx/mx-chain-go/pull/6137/files#diff-44321086a00db9a07d2a7ff50637bf7fad40fd44a6e8ce4794ac05e1bd8a0fb2R130